README.md

```diff
@@ -42,17 +38,17 @@ a new file with ".zonemd" added:
 
 ```
 $ grep ZONEMD vanaheimr.cf.zone.zonemd zonnestelsel.tk.zone.zonemd
-vanaheimr.cf.zone.zonemd:vanaheimr.cf 300 IN ZONEMD 2017122152 1 3858b487874286ff34b47aa9190596bc3c9d29c4
-zonnestelsel.tk.zone.zonemd:zonnestelsel.tk 300 IN ZONEMD 2017120219 1 9dd24e284e7af7141d09d50c6360b80a823e9d54
+vanaheimr.cf.zone.zonemd:vanaheimr.cf 300 IN ZONEMD 2017122152 1 0 3858b487874286ff34b47aa9190596bc3c9d29c4
+zonnestelsel.tk.zone.zonemd:zonnestelsel.tk 300 IN ZONEMD 2017120219 1 0 9dd24e284e7af7141d09d50c6360b80a823e9d54
 ```
 
 You can specify an alternate algorithm with the "-a" flag:
```

Comment:
```suggestion
You can specify an alternate algorithm with the "-a" flag (although currently only SHA384 is supported):
```
We should mention that only one algorithm is now supported, to avoid people pointing this out. 😉